include/openmc/timer.h

```diff
@@ -31,6 +31,7 @@ extern Timer time_event_advance_particle;
 extern Timer time_event_surface_crossing;
 extern Timer time_event_collision;
 extern Timer time_event_death;
+extern Timer time_update_src;
```

Comment:
Why not put this timer on the random ray simulation object?

Reply:
That's possible, but all the timers are currently in the same place, so seemed more organized to just keep them together.

Reply:
Sure thing!